crates/bevy_pbr/src/render/pbr_types.wgsl

```diff
@@ -12,8 +12,14 @@ struct StandardMaterial {
     parallax_depth_scale: f32,
     max_parallax_layer_count: f32,
     max_relief_mapping_search_steps: u32,
+    /// ID for specifying which deferred lighting pass should be used for rendering this material, if any.
+    deferred_lighting_depth_id: u32,
 };
 
+// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+// NOTE: if these flags are updated or changed. Be sure to also update 
+// deferred_flags_from_mesh_material_flags and mesh_material_flags_from_deferred_flags
+// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
How come so few of these flags are used by deferred? Because most are already handled in the deferred prepass so only the ones used by shading need to be propagated?

Reply:
Yes, we only have to keep the flags that are relevant to `fn pbr()` plus fog and minus anything related to alpha blend.